package.json

```diff
@@ -40,7 +40,8 @@
     "ember-auto-import": "^1.10.1",
     "ember-cli-babel": "^7.23.1",
     "ember-cli-htmlbars": "^5.3.1",
-    "ember-decorators": "^6.1.1"
+    "ember-decorators": "^6.1.1",
```

Comment:
We can probably drop `ember-decorators` at this point, right?

Reply:
I think we have to wait til I convert yeti-table proper to drop ember-decorators